src/core/ast.ml

```diff
@@ -319,6 +319,9 @@ and evar = {
 	ev_meta : metadata;
 }
 
+(* TODO: should we introduce CTMono instead? *)
+let ct_mono = CTPath { tpackage = []; tname = "$mono"; tparams = []; tsub = None }
```

Comment:
Let's at least use magic names that are not valid syntax... We might even want to prefix it with the reserved `_hx_` so that macro users cannot sue us.

Reply:
Should we just add `CTMono`?

Reply:
My only concern regarding that is that we run into issues when printing the AST if there's no syntax for it...

Reply:
That already happens sometimes, when we print `Unknown<0>` etc in errors.

Reply:
That's from typing, I'm specifically talking about syntax here which should be isomorphic with the AST.

Reply:
Well, we do print EDisplay _somehow_...

Reply:
Yes but "something isn't ideal so it's fine to introduce other things that aren't ideal" is not a good argument.

Let's go with the hack for the time being and then work towards introducing `CTMono` with a proper syntax a bit later.

Reply:
ok

Reply:
my concern is that people are going to start relying on this hack :)

Reply:
That's why I suggested to use the reserved `_hx_` prefix so we can tell them to GTFO if they use it. :P

Reply:
oh well, `$._hx_mono` is good enough i guess?

Reply:
That looks horrendous, I like it.

Reply:
Ok, done :)

Reply:
People will definitely use this to create monomorphs from macros :)